### ISSUE TITLE
Warn if two datastores report the same dataset name

### ISSUE THREAD

**Issue by philippotto** [scalableminds/webknossos#4463]
See #4458.

**Comment by fm3** [comment]
the backend can acquire and store this information in `updateDataSourceDifferentDataStore` in `DataSetService` – I’m not sure yet what order of reports there leads to this warning being shown but this should be possible.

**Comment by hotzenklotz** [comment]
@fm3 Is this still relevant? WK can now handle DS with the same names.

**Comment by fm3** [comment]
Yes the precedence logic is still kept, this operates on the field now named “directoryName”. One option would be to ingest both datasets with different Ids, but that would break the existing logic.

**Comment by fm3** [comment]
@philippotto since we no longer have many setups with multiple datastores, I’d say the relevance of this is now so small that we can close the issue. Any objections?

**Comment by philippotto** [comment]
yeah, sounds good.
